rwlib/source/loaders/LoaderSDT.cpp

```diff
@@ -116,6 +117,46 @@ char* LoaderSDT::loadToMemory(size_t index, bool asWave) {
         return nullptr;
 }
 
+static int read_packet(void *opaque, uint8_t *buf, int buf_size) {
```

Comment:
Can you make `opaque` a more useful name?

Reply:
Api forces `buf_size` to to be int.

Reply:
About upaque, I don't have better idea.

Reply:
is it a 'handle'?

Reply:
Yeah.

Reply:
If this is a callback the way it's implemented is already a good choice.
There should probably be a comment stating that this is an `// ffmpeg XYZ callback` (whatever XYZ is).

However, when casting the opaque you could give a more descriptive name than `bd` (although it's somewhat self documenting due to the cast)